app/service/page_options_service.rb

```diff
@@ -24,22 +24,17 @@ def all_options_for_answer_type
     options
   end
 
-  def optional
+private
+
+  def hint_options
     [{
-      key: I18n.t("helpers.label.page.answer_type_options.optional"),
-      value: @page.is_optional ? I18n.t("helpers.label.page.answer_type_options.optional_yes") : I18n.t("helpers.label.page.answer_type_options.optional_no"),
+      key: "Hint text",
```

Comment:
There's quite a lot of content in this service file, does it belong in the locale file or is that unnecessary work?